src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -73,6 +73,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' <param name="additionalReferenceDirectories">A string representing additional reference paths.</param>
         ''' <returns>A CommandLineArguments object representing the parsed command line.</returns>
         Public Shadows Function Parse(args As IEnumerable(Of String), baseDirectory As String, sdkDirectory As String, Optional additionalReferenceDirectories As String = Nothing) As VisualBasicCommandLineArguments
+            Debug.Assert(baseDirectory Is Nothing OrElse PathUtilities.IsAbsolute(baseDirectory))
```

Comment:
Looks like this is catching some bad code in `VisualBasicProjectOptionsHelper`. It is passing `""` for `baseDirectory` instead of `Nothing`. That is why so many VB tests are failing now. Really didn't think we'd have this many incorrect usages :frown: